eskip/string_test.go

```diff
@@ -94,6 +94,56 @@ func TestDocString(t *testing.T) {
 		`route2: Path("/some/path") -> "https://www.example.org"`)
 }
 
+func TestPrintNonPretty(t *testing.T) {
+	for i, item := range []struct {
+		route    string
+		expected string
+	} {
+		{
+			"route1: Method(\"GET\") -> filter(\"expression\") -> <shunt>",
```

Comment:
you can also use backtick strings for this, so you don't need to escape doubleqoutes